CrowServe/CrowServe.cpp

```diff
@@ -78,8 +78,17 @@ bool Socket::receive() {
             std::cout << "received message header, type: " << message_header.message_type << " size: " << message_header.message_size << "\n";
             // TODO: figure out protocol-specific message handling
             char buffer[1600];
+            auto p = &buffer;
             auto received = recv(m_socket, &buffer, message_header.message_size, 0);
-            std::cout << "received message of size " << received << "\n";
+            auto sp = std::span<const char>{buffer, (u32)received};
```

Comment:
In this case I typically use `std::span<const char> sp{buffer, (u32)received};` syntax, since it's shorter, and guarantees no accidental copy (compiler isn't required to optimize the auto assign). For the span case this does not matter, since it's a trivially small type.